src/main/daml/Daml/Finance/Data/Time/DateClock/Types.daml

```diff
@@ -1,12 +1,12 @@
 -- Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 -- SPDX-License-Identifier: Apache-2.0
 
-module Daml.Finance.Lifecycle.Types where
+module Daml.Finance.Data.Time.DateClock.Types where
 
 import DA.Time (time)
 import Daml.Finance.Interface.Types.Date.Classes (HasUTCTimeConversion(..))
 
--- | A `Date` which can be converted to `Time` (it is converted to UTC noon).
+-- | A `Date` which can be converted to `Time`. Specifically, each date `D` is mapped to `D 12:00:00 UTC`.
 newtype Unit = Unit Date deriving (Eq)
```

Comment:
Is this type used anymore ?

Reply:
Yup, it's used to convert `Date` to `eventTime` in the clock